Fix #1697: Update migration guide for 1FA activation removal (#1698)
wultra/enrollment-server: docs/PowerAuth-Enrollment-Server-2.1.0.md

<!-- begin box warning -->
The auditing tables may be already updated in your database schema if the database schema is not separated for different PowerAuth applications. In case the column `audit_log.subject_id` and its index `audit_log_subject_id_idx` are already present, you can safely skip this migration step.
<!-- end -->

### Configuration of Activation Removal with 1FA

> The activation removal endpoint `/pa/v4/activation/remove` can be configured for authentication using `POSSESSION` factor (1FA). By default, 2FA authentication is used when removing activations, either `POSSESSION_KNOWLEDGE` or `POSSESSION_BIOMETRY`.

Reviewer: Nitpick
```suggestion
The activation removal endpoint `/pa/v4/activation/remove` can be configured to allow authentication using `POSSESSION` factor (1FA). By default, two-factor authentication is used when removing activations, either `POSSESSION_KNOWLEDGE` or `POSSESSION_BIOMETRY`.
```

> You can enable 1FA authentication for this endpoint by setting the property:

Reviewer: ```suggestion
You can enable 1FA for this endpoint by setting the property:
```

```properties
activation.remove.allow1fa=true
```

> The default value is `false` which disables 1FA authentication.

Author: It's correct.

> The default value is `false` which disables 1FA authentication.

Reviewer: ```suggestion
The default value is `false`, meaning that 1FA is not allowed.
```
